lib/core/presentation/pages/chat/chat_message/widgets/message_item/html_message_widget.dart

```diff
@@ -376,36 +376,17 @@ class MatrixPill extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return InkWell(
-      // onTap: UrlLauncher(outerContext, uri).launchUrl,
-      child: Material(
-        shape: RoundedRectangleBorder(
-          borderRadius: BorderRadius.circular(6),
-          side: BorderSide(
-            color: Theme.of(outerContext).colorScheme.onPrimaryContainer,
-            width: 0.5,
+      //TODO: onTap: UrlLauncher(outerContext, uri).launchUrl,
+      child: Row(
```

Comment:
Add note here why we need to comment out this action. Or remove it.

Reply:
I think adding TODO is enough, the function is explicitly enough, no need to add more info